kaspresso/src/main/kotlin/com/kaspersky/kaspresso/device/screenshots/Screenshots.kt

```diff
@@ -24,4 +33,13 @@ interface Screenshots {
      * @param tag a unique tag to further identify the screenshot. Must match [a-zA-Z0-9_-]+.
      */
     fun takeAndApply(tag: String, block: File.() -> Unit)
+
+    /**
+     * Takes full screenshot if it is possible and applies a function on the file.
+     *
+     * Required Permissions: WRITE_EXTERNAL_STORAGE.
+     *
+     * @param tag a unique tag to further identify the screenshot. Must match [a-zA-Z0-9_-]+.
+     */
+    fun takeFullAndApply(tag: String, block: File.() -> Unit)
```

Comment:
@Areyana I think the word "full" in the names of methods and classes is not clear.What do you think about "fullPicture" or "fullWindow", or "fullScreen", or "fullView"

Reply:
fullScreen i guess fine